### DISCUSSION TITLE
Daily 18/09

### DISCUSSION THREAD

**Discussion by NataliaAlvarezIspc** [grupo-ispc-2023-2-0/workflix-app#59] [Daily]
Con respecto al proyecto:

- ¿Qué hice ayer?
- ¿Qué hice o haré hoy?
- Impedimentos o dificultades

**Comment by NataliaAlvarezIspc** [comment]
El viernes estuve leyendo un poco los apuntes del segundo ejercicio de app moviles. Por la tarde hicimos una videollamada con Brian para que me explique herencia, interfaces y métodos en Java. Pude terminar la segunda parte del primer ejercicio para crear una app y pude hacer la simulación en el celular. El sábado por la mañana tuvimos nuestra weekly. Luego terminé el trabajo práctico de Testing y lo presente. Entre el sábado y el domingo estuve realizando la segunda actividad de app móviles pero no la pude terminar.
Hoy voy a continuar con el segundo ejecicio de app, también haré algunos de los cuestionarios que me faltan de app móviles.
Dificultades: Desconocimiento de los temas que sumados a la mala redacción de los apuntes hace que demore más tiempo.

**Comment by Valentino399** [comment]
Hoy hice el diseño de la aplicación en alta fidelidad con la herramienta de Figma

**Comment by BraianTroncoso** [comment]
El finde estuve repasando conceptos y viendo un poco mas de Java. Estuve viendo nuevamente que patrones de diseño segun el nivel del proyecto podemos adaptar y el que me gustó fue el DAO dado que son solamente consultas directas a la db. También estuve con android en la creacion de un app con conexion a la db tratando de mostrar algo en pantalla lo cual se me complicó un poco. Hoy voy a tratar de listar usuarios.
